src/modelmanager.cpp

```diff
@@ -150,6 +150,14 @@ void processNodeInputs(const std::string nodeName, const rapidjson::Value::Const
     }
 }
 
+void processPipelineInputs(const rapidjson::Value::ConstMemberIterator& pipelineInputsPtr, const std::string& nodeName, std::unordered_map<std::string, std::string>& nodeOutputNameAlias) {
+    for (const auto& pipelineInput : pipelineInputsPtr->value.GetArray()) {
+        SPDLOG_INFO("Alliasing node:{} output:{}, under alias:{}",
+            nodeName, pipelineInput.GetString(), pipelineInput.GetString());
+        nodeOutputNameAlias[pipelineInput.GetString()] = pipelineInput.GetString();
```

Comment:
are we sure here we won't get double time the pipelineInput.GetString()?

Reply:
Possibly we can override value here and not return error?

Reply:
It seems that during checking that we found few issues in other places as well. You can double the inputs/outputs etc. and there is no warning (I think those values are just overridden.

For me those issues are  to be fixed in separate bug.

Reply:
Done:  
https://jira.devtools.intel.com/browse/CVS-43541  
https://jira.devtools.intel.com/browse/CVS-43543  
https://jira.devtools.intel.com/browse/CVS-43544